setup.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+from setuptools import setup
+
+setup (
+    name = 'docker-sigprofiler', 
+    description = 'a implementation for sigprofiler docker CI', 
+    author = 'Jingwei Wang', 
+    url = 'www.example.com', 
+    author_email = 'jw32@sanger.ac.uk', 
+    version = '0.1.0', 
+    python_requires = '>= 3.6', 
+    packages = ['docker_sigprofiler'], 
+    install_requires = [
+        ''
```

Comment:
use it to specify your package dependencies. Key packages in your `requirements.txt` should be here, while packages depended by the key packages don't have to be.